test/bin/mirror_registry.sh

```diff
@@ -29,23 +29,19 @@ get_container_images() {
 prereqs() {
     mkdir -p "${REGISTRY_ROOT}"
     mkdir -p "${REGISTRY_CONTAINER_DIR}"
-    curl -L https://github.com/quay/mirror-registry/releases/download/v${QUAY_VERSION}/mirror-registry-offline.tar.gz -o /tmp/mirror-registry-offline.tar.gz
-    tar xf /tmp/mirror-registry-offline.tar.gz -C "${REGISTRY_ROOT}"
-    rm -f /tmp/mirror-registry-offline.tar.gz
     sudo dnf install -y podman skopeo jq
+    podman run -d -p 5000:5000 --restart always --name registry "quay.io/microshift/distribution:${DISTRIBUTION_VERSION}"
```

Comment:
This command fails if run for the second time. Can we add something like the following to allow running the script more than once without manual intervention?
```suggestion
    podman stop microshift-local-registry || true
    podman run -d -p 5000:5000 --restart always --name microshift-local-registry "quay.io/microshift/distribution:${DISTRIBUTION_VERSION}"
```

Reply:
Added a `rm` too, or else it wont run the second time. Thanks!

Reply:
We might consider mounting path where images are stored on the host, so after kill and run we don't need to recache the images

Reply:
Keep in mind that this procedure happens without knowledge of the RPMs contents. You are going to download and re-push everything everytime because you dont know if something changed.
The container is named to keep the volumes for the next `start` too.

Thinking about this I would be more inclined to include commands to this script. Something along the lines of: setup, start, stop. Only the setup part would try to download and push containers, while the others would be basic operations.

Reply:
I want to believe that `scopeo copy` is smart enough to not do unnecessary work when image exists in dest registry (especially we're using digests, not tags). Also, volumes for `start` don't matter if you `rm` the container.

I'm inclined to tackle this in another PR - for now it's intended for CI, if we want to leverage this on local envs, we should do it some other time

Reply:
> Also, volumes for start don't matter if you rm the container.

In case you want to start the registry because for some reason it was stopped, you wouldnt use this script and would need to do it manually. `mirror_registry.sh` should only be used when needing to mirror all containers. If that was already done, then you should handle it with `podman start/stop`. Thats why I am inclined to include commands.

All this should be included in a separate PR.